standalone/header-footer/src/StandaloneProvider/StandaloneProvider.js

```diff
@@ -20,7 +20,9 @@ export default class StandaloneProvider extends Component {
   }
 
   updateProps(componentProps) {
-    this.setState({ componentProps });
+    this.state.componentProps = { ...this.state.componentProps, ...componentProps };
+
+    this.setState(this.state);
```

Comment:
Instead, this should probably be solved like this:

```
this.setState({ componentProps: { ...this.state.componentProps, ...componentProps }});
```

Reply:
Happy to switch over if thats the preferred way, change incoming 🚀